### PR TITLE
Function docs edits

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Edit the Function docs + update the discord invite link to be displayed as "XYZ invited you to Netdata" where XYZ is a person currently in the Team, previously was a guy I never have met.

#### Issues

1. https://learn.netdata.cloud/docs/top-monitoring/ is cryptic, it is called "Top Monitoring" and inside it talks about Netdata Functions. My intention is to leave it where it is, but change the sidebar label to "Netdata Functions".
2. I don't know how https://learn.netdata.cloud/docs/machine-learning-and-anomaly-detection/run-time-troubleshooting-with-functions got into ML and Anomaly Detection, but it definitely does not belong there. I intend that document to be renamed "Top tab" and be placed inside the "Dashboards and Charts" section of Learn.

@netdata/product and @ilyam8 is the Functions table up-to-date? If not pls update it to display the current updates

> **Note**
>
> I intend to also move these files, but that will depend on if you agree with my intentions here, so it will be a separate PR just for that.

### CODE REVIEW

File: docs/cloud/runtime-troubleshooting-with-functions.md [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
 
-### Prerequisites
+> **Note**
+>
+> If you get an error saying that your node can't execute Functions please check the [prerequisites](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#prerequisites).
 
-The following is required to be able to run Functions from Netdata Cloud.
-* At least one of the nodes claimed to your Space should be on a Netdata agent version higher than `v1.37.1`
-* Ensure that the node has the collector that exposes the function you want enabled ([see current available functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md#what-functions-are-currently-available))
+The main view of this tab provides you with -depending on the Function- two elements: a visualization on the top and a table on the bottom.
```
Comment (ilyam8) [inline docs/cloud/runtime-troubleshooting-with-functions.md:14]: > -depending on the Function- two elements

Typos?

**Comment by ilyam8** [comment]
@Ancairon I don't understand the purpose of the `# Top tab` docs - a textual description of the UI?

**Comment by Ancairon** [comment]
Dashboards and Charts section before had random documentation that didn't reflect how every tab works, also I didn't understand how every tab worked myself, and I think we should have at least one document explaining how each tab works.

File: docs/cloud/runtime-troubleshooting-with-functions.md (hunk 1 above)
Comment (Ancairon) [reply docs/cloud/runtime-troubleshooting-with-functions.md:14]: maybe I need to encase it in parenthesis instead, we do this in Greek, but maybe looks odd in english

**Comment by Ancairon** [comment]
something got strange with the rebase

File: docs/cloud/netdata-functions.md [hunk 2]
```diff
@@ -43,36 +31,20 @@ functions - [plugins.d](https://github.com/netdata/netdata/blob/master/src/colle
 | Systemd-services   | System resource utilization for all running systemd services: CPU, memory, and disk IO.                                                                        | `systemd-cgtop`                 | no            | [cgroups](https://github.com/netdata/netdata/tree/master/src/collectors/cgroups.plugin#readme)                 |
 | Streaming          | Comprehensive overview of all Netdata children instances, offering detailed information about their status, replication completion time, and many more.        |                                 | yes           |                                                                                                            |
```
Comment (hugovalente-pm) [inline docs/cloud/netdata-functions.md:32]:
```suggestion
| Streaming          | Comprehensive overview of all Netdata children instances, offering detailed information about their status, replication completion time, and many more.        |                                 | yes           |                                                                                                            |
| Netdata-api-calls | Real-time tracing of API calls made to the Netdata Agent. It provides information on query, source, status, elapsed time, and more. |                               | no |   |
```

Comment (hugovalente-pm) [reply docs/cloud/netdata-functions.md:32]: @Ancairon also not sure if `Streaming` function needs Cloud. have you confirmed it?

Comment (Ancairon) [reply docs/cloud/netdata-functions.md:32]: I think @ilyam8 might know, I am not sure myself

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**